### PR TITLE
Enable click to scroll

### PR DESCRIPTION
**Pull request by rockia**
Add basic scroll to view feature to the home page

<a href="https://gitpod.io/#https://github.com/mikenikles/webstonehq-com/pull/13"></a>

### CODE REVIEW

File: src/lib/section.svelte [hunk 1]
```diff
@@ -1,10 +1,11 @@
 <script lang="ts">
   let clazz: string = "";
+  export let id;
```
Comment (rockia) [inline src/lib/section.svelte]: I can't just pass `id` into your `<Section>` component and keep it type safe. Consider this is a id pass thru.

File: src/lib/down-arrow.svelte [hunk 2]
```diff
@@ -1,5 +1,18 @@
+<script>
+	export let onClickAction = () => {};
```
Comment (rockia) [inline src/lib/down-arrow.svelte]: Maybe it's OK to not define this empty function here and keep it undefined. Wasn't really thinking too much about this part when I started it.

**Comment by mikenikles** [comment]
Thanks Rock for the idea with the `scrollIntoView` function 💡

What do you think of moving it to the `down-arrow.svelte` component? I think it cleans up the `index.svelte` quite nicely. See e9cb9c1.

**Comment by rockia** [comment]
@mikenikles Yes, that's definitely a nice cleanup. When I first started working on it, I am not sure how reusable the down arrow was so I assigned it to accept generic actions. This is definitely a good move for the use case in this website.

**Pull request merged**